src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application.UnitTests/Services/QueryStringParameterBuilderTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using SFA.DAS.ApprenticeAan.Api.Models;
+using SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+using SFA.DAS.ApprenticeAan.Application.Common;
+using SFA.DAS.ApprenticeAan.Application.Services;
+
+namespace SFA.DAS.ApprenticeAan.Application.UnitTests.Services;
+
+public class QueryStringParameterBuilderTests
+{
+
+    [Test, AutoData]
+    public void Operator_PopulatesModelFromParameters(Guid memberId, DateTime? fromDate, DateTime? toDate,
```

Comment:
Instead of one big test, we need a test per field with both cases where value is null and not null. This will improve your coverage as well.